packages/phoenix-ng/projects/phoenix-ui-components/package.json

```diff
@@ -13,7 +13,8 @@
     "dist"
   ],
   "scripts": {
-    "prepublishOnly": "ng build phoenix-ui-components --prod"
+    "prepublishOnly": "node --eval \"console.error('ERROR: Trying to publish a package that has been compiled by NGCC. This is not allowed.\\nPlease delete and rebuild the package, without compiling with NGCC, before attempting to publish.\\nNote that NGCC may have been run by importing this package into another project that is being built with Ivy enabled.\\n')\" && exit 1",
```

Comment:
All cool apart from this. :)

Reply:
I meant that you need to discard this file completely (like in the previous PR #186) and not have these changes. For making use of lerna, we have to use this `package.json` for publishing but NGCC blocks that so this file should not be changed at all.